app/controllers/pledge_controller.rb

```diff
@@ -0,0 +1,40 @@
+class PledgeController < ApplicationController
+  before_action :find_pregnancy, only: [:create]
+  before_action :find_pledge, only: [:update]
+
+  def create
+    @pledge = @pregnancy.pledges.new(pledge_params)
+    @pledge.created_by = current_user
+    @pledge.save
```

Comment:
I think you can strip this line -- `@pledge.save` will run as part of the conditional just below this, so no need to do this twice. Thoughts?